collections/_members/alonso-lopez.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: person
+name: Alonso López Romo
+Bio:  
```

Comment:
```suggestion
Bio:  I am a graduate of Computational Systems Engineer from Hermosillo's Tech Institute starting the new millennium, and a couple of years later,  I joined Hermosillo’s Tech University, where I have been a faculty member in IT Engineer  programs since. I have been teaching and getting involved in academic events and activities, but at the same time, getting my institution into making bonds up  close with local and regional IT organizations, from all types and contexts.  In middle of this time, I achieved a Master degree in Education. In 2016, I founded an academic IT group, ConSolTI, for which I am the main coordinator, allowing the members, myself included, to manage and develop various projects and collaborations that also will have a direct impact on our IT education programs. So I have been prototyping and developing several solutions for my main local market, but also helping and advising organizations that requested assistance personally, or via our education institution. But one of my my most cherished milestone has been contributing with Layer5 and their project Meshery, in which I have been involving with the most welcoming community and the greatest talented people I know. Beside all this, I have been a professional cellist for about 20 years too, playing violoncello in the Sonora’s Philharmonic Orchestra, and several ensembles in the area; Also, I had the honor of making the world premiere of “Ceremonial” (feb/2020) by composer Tetsuya Hori, and "Fantasía Num.1 para violoncello”(dic/2018) by composer Abdel Vega. So multimedia, arts, entrepreneurship, creative work, video, films, animation, and a big etcetera, are also my passions.

```

Reply:
@alonsolopezr those are some great things that you have under your hood. But, do you mind making it a bit crisp and shorter?

Reply:
Of course, let me do that today.. how many lines or characters would be considered "good"? thanks

Reply:
Hi @Nikhil-Ladha, Would this be short enough? o do you need this text shorter? Thanks. :
I have been a faculty member in IT Engineer programs, since I joined Hermosillo’s Tech University almost 20 years ago. Mainly teaching and getting involved in academic events and activities, also making bonds up close with local and regional IT organizations. In 2016, I founded an academic IT group, for which I am the main coordinator, to develop various projects and collaborations that will benefit our IT programs. Besides being prototyping and developing solutions, I’ve been helping and advising organizations of my local market. But one of my most cherished milestones has been contributing with Layer5’s Meshery, in which I‘ve been involving with the most welcoming community and the greatest talented people I know. Also I’m a professional cellist for about 20 years, playing cello in my state orchestra, and several ensembles. I had the honor of premiering “Ceremonial” (feb. 2020) by composer T. Hori, and "Fantasía Num.1 para violoncello” (dec. 2018) by composer A. Vega. So I’m also passioned about multimedia, arts, entrepreneurship, creative work, video, film and animation.

Reply:
Yes, this would do.
Thanks @alonsolopezr 👍🏼

Reply:
Thank you! Have a great day...